manager-bundle/src/Resources/skeleton/config/security.yml

```diff
@@ -11,10 +11,6 @@ security:
             algorithm: auto
 
     firewalls:
-        dev:
-            pattern: ^/(_(profiler|wdt|error)|css|images|js)/
```

Comment:
Wouldn't it still make sense to keep `(_(profiler|wdt|)` only for the `dev` environment (if that's possible) as these routes are always present in that case?

Reply:
See initial description:

> Therefore, the dev patterns already do not have a firewall anyway.

Reply:
Right 😁